python-package/xgboost/core.py

```diff
@@ -1665,8 +1664,9 @@ def predict(
             feature_names are the same.
 
         training :
-            Whether the prediction value is used for training.  This can effect
-            `dart` booster, which performs dropouts during training iterations.
+            Whether the prediction value is used for training.  This can effect `dart`
```

Comment:
@hcho3 Can this one help?

Reply:
No, this docstring is not as direct as the note being deleted. "This can effect" is too ambiguous, whereas `If you want to obtain result with dropouts, provide training=True.` is direct and clear.

Reply:
@hcho3 I added the note under the `training` parameter.